api/database/seeds/DatabaseSeeder.php

```diff
@@ -14,8 +14,9 @@ public function run()
     {
         Model::unguard();
 
-        $this->call('InvoicesTableSeeder');
+        //$this->call('InvoicesTableSeeder');
```

Comment:
@itech-ro I believe we don't have to comment out code, as we can always go back and see changes.